### PR TITLE
Community/sponsors page

### LINKED ISSUE #3737: Show sponsors on the website
These companies make annual monetary contributions of $420:

- Status Hero
- LGTM
- GoJS

These companies donate services for Shields:

- Heroku
- Sentry
- Nodeping

Four others give us services for free as part of their free offerings (or as part of their free offerings for open-source projects):

- GitHub
- CircleCI
- Cloudflare
- Discord
- **Added:** Coveralls

I'd like to acknowledge all of these entities on Shields.io. I'm thinking to start with, I'd like to make a sponsors page that's linked to from the bottom of the main page, and put everyone's logos there.

In the future I could see have higher tiers for lead sponsors, and maybe companies paying more could have their logos at the bottom of the home page (or for that matter, the top of the home page). But a sponsor page with everyone's logos seems like a good place to start.

### PR DESCRIPTION
**Pull request by chris48s**
Closes #3737

First bash at a community page on shields.io thanking our contributors and financial donors (I think its good to frame all of this as "community" rather than separating out financial sponsors). This pulls together some of the content from the README and the suggestions from https://github.com/badges/shields/issues/3737 into one place and I've tried to do that in a relatively cohesive way.

I'm fairly keen to:

a) Lean on OpenCollective's handy SVG images as much as possible so this doesn't require too much maintenance
b) Avoid duplicating the same content in two places

### CODE REVIEW

File: frontend/components/footer.tsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ export default function Footer({ baseUrl }: { baseUrl: string }): JSX.Element {
       </p>
 
       <p>
+        <SpacedA href="/community">Community</SpacedA>
```
Comment (chris48s) [inline frontend/components/footer.tsx:76]: I've linked this in the footer. Maybe it should also be somewhere more prominent, but I'm not sure where..

Comment (calebcartwright) [reply frontend/components/footer.tsx:76]: I think this is probably the best place for this for now :+1:

**Comment by calebcartwright** [comment]
I haven't had a chance to go through the code changes yet, but did deploy a review app and gave it a spin. It looks great!

File: frontend/pages/community.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        🙏 This project exists thanks to all the nice people who contribute
+        their time to work on the project.
+        <p>
+          <object data="https://opencollective.com/shields/contributors.svg?width=600" />
+        </p>
+      </SponsorContainer>
+
+      <SponsorContainer>
+        ✨ Shields is helped by these companies which provide their products or
+        services to open source projects for free:
```
Comment (PyvesB) [inline frontend/pages/community.tsx:97]: Would LGTM and Uptime Robot fit in this section as well?

Comment (chris48s) [reply frontend/pages/community.tsx:97]: Yeah this is a good point. I added the stuff listed in https://github.com/badges/shields/issues/3737 but there are probably a bunch of other things where we're using some kind of free plan.. netlify fits into this category as well. Can you think of anything else?

LGTM is already listed as a sponsor, but we do use their service too.

Comment (chris48s) [reply frontend/pages/community.tsx]: I've added the ones we've thought of. If more come up, we can always add them later..

**Pull request merged**